src/constants/constants.ts

```diff
@@ -76,6 +76,8 @@ export const REDIRECTION_SERVER_IPS = [
   "18.138.108.8",
   "18.139.47.66",
 ]
+export const ALLOWED_DNS_ERROR_CODES = ["ENOTFOUND", "ENODATA"]
+
```

Comment:
Hmm `ENOTFOUND` seems to be a catch all actually, should we be including this in the list of acceptable error codes?

From the node dns docs:
```
Keep in mind that err.code will be set to 'ENOTFOUND' not only when the host name does not exist but also when the lookup fails in other ways such as no available file descriptors.
```

Reply:
do you know of any other way to capture no host name ah?
i agree that this is not the best way since we have a blind spot here, but their api does not seem to have an check for just a host name. 

note that do have an error code for say timeout (`ERR_SOCKET_CONNECTION_TIMEOUT `), so was hopeful that this would enough to catch transient network errors

Reply:
Does this happen on every site launch? Or is this an infrequent error caused by the agency not putting up their records in time? If it's infrequent there might be value in having to manually check, rather than possibly giving the agency a false positive

Reply:
This check happens once every site launch. But the error 'ENOTFOUND' is specifically thrown for every new domain that is being launched. We have had multiple incidents happen due to human error during SL (think one happened just this week with the pr to indirection layer iirc), so I am very inclined not to have manual checks. This PR's test plan has a sense check by running dns.promises on your docker for both resolve6 AND CAA for sites that have and dont have the above values. My stance is errors slip through will be rare, and as such the false positive is low enough to not affect operations.